src/components/MoneyRequestConfirmationList.js

```diff
@@ -228,6 +229,17 @@ function MoneyRequestConfirmationList(props) {
         [props.session.email],
     );
 
+    /**
+     * Navigate to details page of selected user
```

Comment:
```suggestion
     * Navigate to profile of selected user
```

Reply:
Oh, sorry for that! I didn't remember to ammend the comment